etc/before-remove.sh

```diff
@@ -28,11 +34,18 @@ fi
 # Disable NSS plugin.
 sed -i "s/ google//" /etc/nsswitch.conf
 
-# Disable AuthorizedKeysCommand and restart sshd.
+# Disable AuthorizedKeysCommand.
 sed -i "s#AuthorizedKeysCommand ${DIR}/authorizedkeys##" /etc/ssh/sshd_config
 sed -i "s/AuthorizedKeysCommandUser ${ACCOUNT}//" /etc/ssh/sshd_config
```

Comment:
Shouldn't this be AuthorizedKeysCommandUser in once case and AuthorizedKeysCommandRunAs in another case?